From a42c44c42353eac6d076f9e226ce3dba9ff02622 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Wed, 6 Sep 2017 20:56:34 +0800
Subject: [PATCH 1260/5242] MLK-17491-17 dts: imx7ulp: add ipg clock for tpm
 timer

commit  593802352bddd36278dc2a7d9bc51468ef05d6e5 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to reference mannual, there should be also an ipg clock,
so add it.

Cc: Shenwei Wang <shenwei.wang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 3269110..f9b7de0 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -176,7 +176,9 @@
 			compatible = "fsl,imx7ulp-tpm";
 			reg = <0x40260000 0x1000>;
 			interrupts = <GIC_SPI 22 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&clks IMX7ULP_CLK_LPTPM5>;
+			clocks = <&clks IMX7ULP_CLK_LPTPM5>,
+				 <&clks IMX7ULP_CLK_NIC1_BUS_DIV>;
+			clock-names = "per", "ipg";
 			assigned-clocks = <&clks IMX7ULP_CLK_LPTPM5>;
 			assigned-clock-parents = <&clks IMX7ULP_CLK_FIRC>;
 		};
-- 
1.7.9.5

